src/uta/tools/eutils.py

```diff
@@ -0,0 +1,30 @@
+import requests
+
+from uta import EutilsDownloadError
+
```

Comment:
It might be worth creating a enumeration for the two supported file formats. Something like ...

```
import enum from Enum

class NcbiFileFormatEnum(str, Enum):
    fasta = "fasta"
    genbank = "gb"
```

Then this method could use `NcbiFileFormatEnum` inplace of `str`. And you would not need the check to see if the input strings are supported.